Xamarin.Forms.ControlGallery.iOS/AppDelegate.cs

```diff
@@ -162,6 +162,8 @@ public override bool FinishedLaunching(UIApplication uiApplication, NSDictionary
 			// Forms.SetFlags("CollectionView_Experimental", "Visual_Experimental", "Shell_Experimental");
 			Forms.Init();
 			FormsMaps.Init();
+			FormsMaterial.Init();
```

Comment:
We might want to do this for all the other app projects as well - the pages and embedded I think